int32 id

# track location
float64 s_start
float64 s_end

# map locations
float64 start_x
float64 end_x
float64 start_y
float64 end_y

# information
string side
bool possible_trailing
float64 score
int32 attempts


